app/main.py

```
@@ -763,5 +786,221 @@ def list_flows_cmd(
+    force: bool = typer.Option(
+        False, "--force", "-f",
+        help="Overwrite if a schedule with this name already exists."
+    ),
+) -> None:
+    """Validate and copy a schedule YAML into ``.atelier/schedules/``."""
+    store = _schedule_store()
+    try:
+        dest = store.install(file, force=force)
+    except FileExistsError as e:
+        console.print(f"[red]{e}[/red]  [dim](use --force to overwrite)[/dim]")
+        raise typer.Exit(code=1)
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)